dataStorage/storage.py

```diff
@@ -1,9 +1,9 @@
-def print_storage():
-    print("Hello from storage")
 class dataMaintainer:
     def __init__(self, argument1, argument2):
         return
+    def __init__(self):
+        return
 
-    def print_storage():
+    def __print_storage(self):
```

Comment:
Note that private functions cannot be publicly used outside of the scope of the class. This will need to change to a public access modifier.